oneanddone/tasks/forms.py

```diff
@@ -25,9 +25,11 @@ def __init__(self, *args, **kwargs):
 
 
 class FeedbackForm(forms.ModelForm):
+    completion_time_in_minutes = forms.IntegerField(
+                      label= _('How many minutes did the task take to complete?'))
```

Comment:
We ask for feedback both for completed attempts and also for abandoned attempts, so this label is not always going to be accurate. I suggest changing it to `How many minutes did you spend on the task?`